src/components/search-concept/search-concept.component.tsx

```diff
@@ -46,7 +53,17 @@ export const SearchConcept: React.FC<SearchConceptProps> = ({
     }
   };
 
-  const debounceFn = _debounce(onSearch, 500);
+  const debouncedSearch = useRef(
+    _debounce(async (search) => {
+      search != "" && (await onSearch(search));
+    }, 400)
```

Comment:
`400` milliseconds might not be large enough.

Reply:
Also, Could you make this more readable?

Reply:
it tested this seems enough, previously debounce setup was different